src/app/session-lobby/session-lobby.component.ts

```diff
@@ -8,14 +10,21 @@ import { TeacherSessionService } from '../teacher-session.service';
 })
 export class SessionLobbyComponent implements OnInit {
 
-  sessionID: string;
 
-  constructor(public teacherSessionService: TeacherSessionService) {
-    this.sessionID = 'demo-session'; // Temporary until a way to get the session is implemented
-  }
+  sessionID = 'demo-session'; // Temporary until a way to get the session is implemented
+  roundData = {flowerSpeciesIds: ['asclepias_syriaca', 'coreopsis_palmata'],
+                        status: 'fine',
```

Comment:
I know we haven't defined what a round `status` is, but what does "fine" mean here?

Reply:
Oh, that was supposed to be a blatant placeholder—something that obviously isn't production data 😛